data/sql/updates/pending_db_world/rev_1695820418952974700.sql

```diff
@@ -0,0 +1,7 @@
+-- #12145 midsummer add spell script spell_midsummer_ribbon_pole_visual
+DELETE FROM acore_world.spell_script_names WHERE spell_id IN (29531, 29705, 29726, 29727);
+INSERT INTO acore_world.spell_script_names (spell_id, ScriptName) VALUES
```

Comment:
```suggestion
DELETE FROM `spell_script_names` WHERE `spell_id` IN (29531, 29705, 29726, 29727);
INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
```

you don't need to specificy the table, as it is already use when adding a file to `pending_db_world` also you should use AC sql conventions

Reply:
thanks for the headsup, I generated them with DBeaver but should definitely follow AC convention